tests/unit-tests/dds/DCPS/security/AccessControlBuiltInImpl.cpp

```diff
@@ -4,11 +4,18 @@
 #include <dds/DCPS/security/TokenWriter.h>
 #include <dds/DCPS/Service_Participant.h>
 
-#include <dds/DdsDcpsC.h>
+#include "ace/config-lite.h"
+
+#ifdef ACE_HAS_CPP11
+#  define HAS_STD_SHARED_PTR
+#endif
 
+#include <dds/DdsDcpsC.h>
+#ifdef HAS_STD_SHARED_PTR
+#else
```

Comment:
```suggestion
#ifndef HAS_STD_SHARED_PTR
```

Reply:
That was a mistake. I neglected an include that was intended for the ifdef case.